uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/PublicHolidayTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+
+/**
+ * Class CRM_HRLeaveAndAbsences_BAO_PublicHolidayTest
+ *
+ * @group headless
+ */
+class CRM_HRLeaveAndAbsences_BAO_PublicHolidayTest extends CiviUnitTestCase implements HeadlessInterface {
```

Comment:
Since you're not using anything from CiviUnitTestCase, you can extend PHPUnit_Framework_TestCase instead and your tests will run MUCH faster.

If you do that, you won't need the $_tablesToTruncate property, neither the setUp nor the tearDown methods. Also, you'll need to implement the TransactionalInterface to make sure the date created by a test will be deleted at its end.